src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -99,8 +99,8 @@ def _filter_nones(some_dict: Dict[str, Any]) -> Dict[str, Any]:
     return {key: value for key, value in some_dict.items() if value is not None}
 
 
-class PartialSnapshot:
-    def __init__(self, snapshot: Optional["Snapshot"] = None) -> None:
+class NewSnapshot:
```

Comment:
Tbh I don't like the name, what about `RealSnapshot` or `EnsSnapshot`?

Reply:
This was just temporary to not remove old snapshot before the new one was working. I think just `Snapshot` works well.